### ISSUE TITLE
Option to have more than 8 apps 

### ISSUE THREAD

**Issue by carlosdeque-s** [tanujnotes/Olauncher#184]
Does somebody know a simple coding way to change the maximul number of apps available in the desktop screen? 

Which is a simple way to implement that only on my launcher?

**Comment by carlosdeque-s** [comment]
> Does somebody know a simple coding way to change the maximul number of apps available in the desktop screen?
> 
> Which is a simple way to implement that only on my launcher?

Would adding new lines with more n° of apps options do it? 


![Screenshot_20220518-001357_Code Editor](https://user-images.githubusercontent.com/72588466/168920420-24c17b7a-827e-42bc-bb93-086032d8a199.jpg)

**Comment by tanujnotes** [comment]
Yes but you'll have to do it in a few different places. Check Prefs.kt file as well.

**Comment by carlosdeque-s** [comment]
Ok, I'll try it thanks

**Comment by carlosdeque-s** [comment]
Is it a design intention to have 8 apps as the maximum number? If not I could make a simpler way to have as many apps on the launcher as you want while at the same time making the code simpler. Would you like it?

**Comment by tanujnotes** [comment]
I'd welcome a simpler way.
There are 2 reasons for having max 8 apps - minimlism and smaller screens. Having more than 8 apps means we'll have to add scrolling on the home screen as well. I don't want that especially when app drawer is just a swipe away. 8 + 2 swipe left/right apps works fine for most people.

**Comment by carlosdeque-s** [comment]
Hadn't thought of scrolling, just limit the max number of apps available to the user in function of the resolution of their screen and the size of the font used. 

Having said that now I'm through exams (Industrial engineering) and would not want to put myself through this until July (when I end them). Then I will be very happy to work on it so I can try github.

I also wanted to say that your app is amazing and I love it. Right now I'm using it with 11 apps but just copying and pasting repeatedly the code I was refering to at the beggining of this thread.

**Comment by tanujnotes** [comment]
As I said, the current setup works well for most people so changing anything is not a priority. It started with just 4 apps and I kept copy pasting.
If you want to work on a feature, translations in different languages and ability to resize text is highest priority.
